Command/AlignCommand.php

```diff
@@ -55,6 +58,30 @@ protected function configure()
                 InputArgument::REQUIRED,
                 'Environment target name'
             )
+            ->addArgument(
+                'scrollSize',
+                InputArgument::OPTIONAL,
+                'Size of the elasticsearch scroll request',
+                self::DEFAULT_SCROLL_SIZE
+            )
+            ->addArgument(
+                'scrollTimeout',
+                InputArgument::OPTIONAL,
+                'Time to migrate "scrollSize" items i.e. 30s or 2m',
+                self::DEFAULT_SCROLL_TIMEOUT
+            )
+            ->addArgument(
+                'contentType',
+                InputArgument::OPTIONAL,
+                'The content type you wish to align'
+            )
+            ->addOption(
```

Comment:
ContentType can be ignored as

1. It's not longer exist in elasticsearch 7

1. We can now filter on the _contenttype field ( so via the searchQuery attribute)